src/Features/Onboarding.php

```diff
@@ -120,6 +133,26 @@ private function add_actions() {
 		add_action( 'current_screen', array( $this, 'redirect_old_onboarding' ) );
 	}
 
+	/**
+	 * Trigger the woocommerce_onboarding_profile_completed action
+	 *
+	 * @param array $old_value Previous value.
+	 * @param array $value Current value.
+	 */
+	public function trigger_profile_completed_action( $old_value, $value ) {
+		if ( ! isset( $value['completed'] ) || ! $value['completed'] ) {
```

Comment:
Should we also check the old value here to make sure it was not complete so we don't trigger this action on updating other properties if the profiler was already complete?

Reply:
done